include/zephyr/mgmt/hawkbit/hawkbit.h

```diff
@@ -33,6 +33,10 @@
 enum hawkbit_response {
 	/** matching events were not received within the specified time */
 	HAWKBIT_NO_RESPONSE,
+	/** an update was installed. Reboot is required to apply it */
+	HAWKBIT_UPDATE_INSTALLED,
```

Comment:
Change to _DOWNLOADED, as event enum "HAWKBIT_EVENT_UPDATE_DOWNLOADED". Update comment.